### PR TITLE
Add Realtime lane (manual trigger)

### PR DESCRIPTION
**Pull request by jordigilh**
Adds a new manual lane for k8s 1.22 (root) to run only the [functional tests](https://github.com/kubevirt/kubevirt/blob/280697044fcdaf163bf16b99ef4cdb6528950bb3/tests/realtime/realtime.go) for realtime use cases. This PR takes into account the changes introduced in this other [PR](https://github.com/kubevirt/kubevirt/pull/6182) (kubevirt) to the [automation/test.sh](https://github.com/kubevirt/kubevirt/blob/280697044fcdaf163bf16b99ef4cdb6528950bb3/automation/test.sh) script to tune the created cluster to support realtime VMs.

@fgimenez @vladikr @dhiller @pkliczewski can you take a look?

### CODE REVIEW

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml
Comment (fgimenez) [inline github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml:1615]: No need for this preset, it is only required on windows or RHEL lanes

**Review by fgimenez — COMMENTED** [review]
Looks good! small nit, other than that it is ready to go once the kubevirt/kubevirt changes land

**Review by fgimenez — APPROVED** [review]
/approve
/hold

Thanks! feel free to unhold when ready

**Comment by jordigilh** [comment]
> /approve
> /hold
> 
> Thanks! feel free to unhold when ready

Do I need to wait until the [PR](https://github.com/kubevirt/kubevirt/pull/6182) in kubevirt is merged before this one can be merged? Or can I merge this one and trigger the test manually in the kubevirt PR to validate that the lane works fine?

**Comment by fgimenez** [comment]
> Do I need to wait until the [PR](https://github.com/kubevirt/kubevirt/pull/6182) in kubevirt is merged before this one can be merged? Or can I merge this one and trigger the test manually in the kubevirt PR to validate that the lane works fine?

As you see fit, no need to wait given that the job is not triggered by default. This one can be merged with `/unhold` in a comment

**Pull request merged**
